third_party/mbedtls/native_its/src/file/its_file.c

```diff
@@ -0,0 +1,384 @@
+/*
```

Comment:
I didn't dig into that but mbedtls comes with MBEDTLS_PSA_ITS_FILE_C, do we need native file based impl despite of that?

Reply:
Yeah.. it's pretty basic and non configurable (e.g. w.r.t. paths).